components/shared/Play/Play.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import Icon from "../Icon/Icon";
+import "./Play.module.scss";
+
+interface PlayProps {
+  onClick: () => void;
+}
```

Comment:
The difference between `interface` and `type`, we can overload and extend `interface`. `type` its just static types. As far, as we pass regular button attributes in properties. I'd recommend update one to `type`

Reply:
Can you please pass all button attributes in types? 🙏 You can use `ButtonHTMLAttributes` here